actors/stage/src/main/java/com/ea/orbit/actors/runtime/Messaging.java

```diff
@@ -69,8 +70,8 @@
     private ClusterPeer clusterPeer;
     private Execution execution;
     private final AtomicInteger messageIdGen = new AtomicInteger();
-    private Map<Integer, PendingResponse> pendingResponseMap = new ConcurrentHashMap<>();
-    private PriorityBlockingQueue<PendingResponse> pendingResponsesQueue = new PriorityBlockingQueue<>();
+    private final Map<Integer, PendingResponse> pendingResponseMap = new ConcurrentHashMap<>();
+    private final PriorityBlockingQueue<PendingResponse> pendingResponsesQueue = new PriorityBlockingQueue<>(50, new PendingResponseComparator());
```

Comment:
50 seems like a reasonable number for initial capacity (required to use the comparator constructor), default in JDK source is 11..